selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -428,24 +469,27 @@ public void scrollAndSelectItem(String path) {
    * @param path
    */
   public void openItemByPath(String path) {
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
     waitItem(path);
-    String locator = "//div[@path='/%s']/div";
-    WebElement item =
-        redrawUiElementsWait.until(
-            ExpectedConditions.visibilityOfElementLocated(By.xpath(String.format(locator, path))));
+    selectItem(path);
+    waitItemIsSelected(path);
+
     try {
-      item.click();
+      getProjectExplorerItem(path).click();
       waitItemIsSelected(path);
-      actionsFactory.createAction(seleniumWebDriver).doubleClick(item).perform();
+      action.moveToElement(getProjectExplorerItem(path)).perform();
+      action.doubleClick().perform();
     }
     // sometimes an element in the project explorer may be is not attached to the DOM. We should
     // refresh all items.
     catch (StaleElementReferenceException ex) {
+      LOG.error(ex.getLocalizedMessage(), ex);
```

Comment:
IMHO it's better to log exception as warn() here so as we don't interrupt the execution of tests.